integration-cli/docker_cli_start_test.go

```diff
@@ -15,9 +16,9 @@ func (s *DockerSuite) TestStartAttachReturnsOnError(c *check.C) {
 	dockerCmd(c, "wait", "test")
 
 	// Expect this to fail because the above container is stopped, this is what we want
-	if _, _, err := dockerCmdWithError("run", "-d", "--name", "test2", "--link", "test:test", "busybox"); err == nil {
-		c.Fatal("Expected error but got none")
-	}
+	_, _, err := dockerCmdWithError("run", "-d", "--name", "test2", "--link", "test:test", "busybox")
+	// Expected error but got none
```

Comment:
This is used to show why the assert failed, and is moved from `c.Fatal`.
I can find a better sentence.